src/collections/service-mesh-books/istio-up-and-running/index.mdx

```diff
@@ -5,19 +5,30 @@ cover: ./istio-up-and-running.webp
 thumbnail: ./istio-up-and-running_top.png
 author: "Lee Calcote and Zack Butcher"
 abstract: "A must-read for anyone looking to get started with Istio."
+description: "Layer5 and HashiCorp join forces to launch a groundbreaking partnership, bringing advanced cloud-native infrastructure solutions for containerized and non-containerized workloads through Meshery and Consul integration. Explore the power of Meshery's support for Consul service mesh, including sample applications, performance management, and experimental support for WebAssembly, demonstrated at DockerCon 2020 for near-native fine-grained traffic control."
```

Comment:
This description is for the Layer5 and HashiCorp partnership, not for the Istio: Up and Running book.

Reply:
Hi @Mohith234 I have fixed the problem @leecalcote pointed. Kindly give it a final look. Thank you.